packages/edit-post/src/index.js

```diff
@@ -155,3 +155,4 @@ export { default as PluginPostStatusInfo } from './components/sidebar/plugin-pos
 export { default as PluginPrePublishPanel } from './components/sidebar/plugin-pre-publish-panel';
 export { default as PluginSidebar } from './components/sidebar/plugin-sidebar';
 export { default as PluginSidebarMoreMenuItem } from './components/header/plugin-sidebar-more-menu-item';
+export { default as __experimentalFullscreenModeClose } from './components/header/fullscreen-mode-close';
```

Comment:
Can this component be shared between `edit-site` and `edit-post`?

Reply:
We considered that in https://github.com/WordPress/gutenberg/pull/20989#discussion_r394387344 but decided not to do it for now.